api/v1beta1/metallb_types.go

```diff
@@ -30,6 +30,8 @@ type MetalLBSpec struct {
 
 	// Foo is an example field of MetalLB. Edit MetalLB_types.go to remove/update
 	MetalLBImage string `json:"image,omitempty"`
+	// +optional
+	SpeakerDaemonNodeSelector map[string]string `json:"nodeSelector,omitempty"`
```

Comment:
I'd call this just SpeakerNodeSelector (same for the api field), seems more straightforward.